src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -49,6 +53,19 @@
      * Used for initial separation of command word and args.
      */
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
+    private static boolean userIsLoggedOn = false;
```

Comment:
i think there could be a error here. 
If AddressBookParser is not initialised, it will be false.
Initialise your boolean userIsLoggedOn in the constructor could be better.